keyboards/zeal60/Makefile

```diff
@@ -0,0 +1,3 @@
+ifndef MAKEFILE_INCLUDED
```

Comment:
This file isn't necessary and should be deleted.

Reply:
fixed